src/utils/uniq.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2015 CANAL+ Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Returns the input array without duplicates values.
+ * All values are unique.
+ */
+export default typeof (window as any).Set === "function" ?
+  /**
+   * @param {Array.<*>} arr
+   * @returns {Array.<*>}
+   */
+  function uniqFromSet<T>(arr: T[]) {
+    return Array.from(new Set(arr));
+  } :
+  /**
+   * @param {Array.<*>} arr
+   * @returns {Array.<*>}
+   */
```

Comment:
Maybe here a single JSDoc for both at the top of the export would be more readable